01-Git_Bash_Make/dz1.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int game() 
+{
+        int d;
+        int r;
+    
+        srand(time(NULL));
+        printf("Enter number 0-9>");
+        scanf("%d", &d);
+        r = rand() % 10;
+        return d == r;
+}
+
+int main(void) 
+{
+   
+        if (game()) {
+                printf("You win\n");
+                return 0;
+        }
+        else {
+                printf("You loose\n");
+                return 1;
```

Comment:
return 1 it a way to say other programs that your program is crashes with error code 1. Please, remove all unnecessary return. Be careful with coding style.